vanilla-project/src/main.ts

```diff
@@ -20,8 +22,22 @@ interface AppStoreInterface {
 const appStore: AppStoreInterface[] = mock;
 
 appStore.forEach(app => {
-  const newDiv: HTMLDivElement = document.createElement('div');
-  newDiv.className = "inner-card";
-  newDiv.textContent = app.name;
-  document.body.appendChild(newDiv);
+  const outterCard = document.getElementById("card");
+
+  if (outterCard !== null) {
```

Comment:
!outterCard

Reply:
raviv right but he meant just: 
```suggestion
  if (outterCard) {
```

this is check if the object null or undefind, if its null it will pass the if statment